scripts/monitoring/cron-send-etcd-status.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python
+'''
+   Command to send status of etcd to zabbix
+'''
+
+#This is not a module, but pylint thinks it is.  This is a command.
+#pylint: disable=invalid-name
+
+from openshift_tools.monitoring.zagg_sender import ZaggSender
+import json
+import requests
+
+def main():
+    ''' Get data from etcd API
+    '''
+
+    API_HOST = 'https://localhost:4001/v2/stats/store'
+    SSL_CLIENT_CERT = '/etc/openshift/master/master.etcd-client.crt'
+    SSL_CLIENT_KEY = '/etc/openshift/master/master.etcd-client.key'
+
+    # Fetch the store statics from API
+    request = requests.get(API_HOST, cert=(SSL_CLIENT_CERT, SSL_CLIENT_KEY), verify=False)
```

Comment:
Right now the ops-runner will fail - but you are right, I will catch that exception and send it as an item, so we can alert high on it if it really fails.